extensions/amp-story-audio-sticker/validator-amp-story-audio-sticker.protoascii

```diff
@@ -0,0 +1,241 @@
+tags: {  # amp-story-audio-sticker
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-story-audio-sticker"
+    version: "0.1"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-story-audio-sticker>
+  html_format: AMP
+  tag_name: "AMP-STORY-AUDIO-STICKER"
+  requires_extension: "amp-story-audio-sticker"
+  mandatory_parent: "AMP-STORY-GRID-LAYER"
+  attr_lists: "extended-amp-global"
+  amp_layout: {
+    supported_layouts: CONTAINER
+  }
+  attrs: {
+    name: "size"
+    value: "small"
+    value: "large"
+  }
+  attrs: {
+    name: "sticker"
+    value: "cat-sticker"
+    value: "tape-player"
+    value: "loud-speaker"
+    value: "audio-cloud"
+  }
+  attrs: {
+    name: "sticker-style"
+    value: "outline"
+    value: "dropshadow"
+  }
+  child_tags: {
+    child_tag_name_oneof: "AMP-STORY-AUDIO-STICKER-PRETAP"
+    child_tag_name_oneof: "AMP-STORY-AUDIO-STICKER-POSTTAP"
+  }
+  descendant_tag_list: {
+    # Allowlisting as many components as possible, unless they could result in a
+    # bad UX.
+    name: "amp-story-audio-sticker-allowed-descendants"
+    tag: "A"
+    tag: "ABBR"
+    tag: "ADDRESS"
+    tag: "AMP-3D-GLTF"
+    tag: "AMP-3Q-PLAYER"
```

Comment:
not much experience in writing these protoascii's but im curious if this is desirable that we allow video players inside an audio sticker?

/cc @banaag for any input

Reply:
Good catch. Removed all player tags.